microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -0,0 +1,96 @@
+---
+description: How to detect Microsoft Edge data with User-Agent Client Hints and User-Agent strings.
+title: Detecting Windows 11 using User-Agent Client Hints
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/15/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, user-agent string, ua string, user-agent client hints, user agent client hints, ua client hints, ua ch, windows 11, win11, detecting windows 11, windows detection
+---
+# Detecting Windows 11 using User-Agent Client Hints
+
+<!--
+Restrict the lexicon to these forms:
+User-Agent string
+user agent string
+User-Agent Client Hints
+-->
+
+Websites can use User Agent information that's sent from the browser to detect information such as browser brand, version, and platform. There are two approaches for sites to access User Agent information:
+*  User-Agent strings (legacy).
+*  User-Agent Client Hints (recommended).
+
+For details about these two approaches, see [Detecting Microsoft Edge from your website](user-agent-guidance.md).
+
+To differentiate between users who are using Windows 11 and Windows 10, starting in Microsoft Edge version 94, User-Agent Client Hints reflects the following header-field values:
+
+| Header field | Value that indicates Windows 10 | Value that indicates Windows 11 |
+| --- | --- | --- |
+| `Sec-CH-UA-Platform` | `Windows` | `Windows` |
+| `Sec-CH-UA-PlatformVersion` | `10.0.0` and above | `13.0.0` and above |
+
+Unfortunately, User-Agent strings won't be updated to differentiate between Windows 11 and Windows 10.  We don't recommend using User-Agent strings to retrieve user agent data.  For browsers which don't support User-Agent Client Hints, sites won't be able to differentiate between Windows 11 and Windows 10.
+
+
+<!-- ====================================================================== -->
+## Browsers that support User-Agent Client Hints
+
+The following table shows which browsers support differentiating between Windows 11 and Windows 10.
```

Comment:
@jenny-microsoft - Can you use fewer than 3 different values?  Currently the column has several values; unclear:
`-`
`None`
`N/A`

Reply:
@jenny-microsoft - Yes, I think inserting that text directly would be more helpful, since there are 3 similar looking notations now.

Reply:
We could also just remove Safari from the list. It's interesting to list it in the context of broader UA Client Hints support, but for a page dedicated to differentiating Win10 vs. Win11 it's unncessary.

Reply:
Resolution: Removed the now-empty "Alternative" column.